ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-eus-4.14.yaml

```diff
@@ -110,6 +110,25 @@ tests:
           memory: 200Mi
     - chain: openshift-upgrade-qe-test-eus
     workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
+- as: gcp-ipi-workers-rhel8-fips-f28
+  cron: 9 13 12 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE
+        name: release:intermediate
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-sanity-eus
```

Comment:
should be `openshift-upgrade-qe-test-eus`, right?

Reply:
Right, because of pj-rehearse limit, I am using the sanity chain to run the testing, once completed, will switch back to `openshift-upgrade-qe-test-eus`.

Reply:
Done.